src/main/java/edu/kis/powp/jobs2d/events/SelectLoadMacroOptionListener.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+import edu.kis.powp.jobs2d.features.MacroFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.List;
+
+public class SelectLoadMacroOptionListener implements ActionListener {
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        List<DriverCommand> commands = MacroFeature.getMacroDriver().getMacro();
```

Comment:
Please remove dependency with MacroFeature (you can store macroDriver as a field).